### ISSUE TITLE
You should add relu to CNN_OriginalAvg

### ISSUE THREAD

**Issue by sugaruncle** [FedML-AI/FedML#96]
YOU! FORGET! ReLU!
![image](https://user-images.githubusercontent.com/35253472/105810690-61a12680-5fe6-11eb-99e3-d7958b59adae.png)
There should be relu
https://github.com/FedML-AI/FedML/blob/master/fedml_api/model/cv/cnn.py
![image](https://user-images.githubusercontent.com/35253472/105810794-8b5a4d80-5fe6-11eb-9a05-c0403ef34063.png)

**Comment by chaoyanghe** [comment]
https://stackoverflow.com/questions/35543428/activation-function-after-pooling-layer-or-convolutional-layer
MaxPool(Relu(x)) = Relu(MaxPool(x)) 

https://www.tensorflow.org/federated/tutorials/tff_for_federated_learning_research_compression
model = tf.keras.models.Sequential([
      tf.keras.layers.InputLayer(input_shape=(28, 28, 1)),
      conv2d(filters=32),
      max_pool(),
      conv2d(filters=64),
      max_pool(),
      tf.keras.layers.Flatten(),
      tf.keras.layers.Dense(512, activation=tf.nn.relu),
      tf.keras.layers.Dense(10 if only_digits else 62),
      tf.keras.layers.Softmax(),
  ])
  
  The above model definition is from the Google TF team, which is also the original author of the FedAvg algorithm.

**Comment by sugaruncle** [comment]
There are Relu in your second link 
https://www.tensorflow.org/federated/tutorials/tff_for_federated_learning_research_compression
![image](https://user-images.githubusercontent.com/35253472/105942334-003a8f80-609a-11eb-822d-28bea6bd0d6c.png)

**Comment by chaoyanghe** [comment]
oh, I see. Thanks. Could you please help to pull request this fix?

**Comment by sugaruncle** [comment]
OK, I have pulled the request.
